fuseki/sample-ubd.trig

```diff
@@ -82,13 +83,39 @@ graph <cube-project/ubd/dimensions-metadata> {
 
   <cube-project/ubd/dimensions-metadata/pollutant>
     schema:about <https://environment.ld.admin.ch/foen/ubd/28/pollutant> ;
+    scale:scaleOfMeasure scale:Nominal ;
+    cc:dimensionMapping <cube-project/ubd/dimension-mapping/pollutant> ;
```

Comment:
This would be created when a dimension is set to `Nominal`

Reply:
You mean when a mapped managed dimension is selected right? Not all Nominal dimensions will be mapped

Reply:
> You mean when a mapped managed dimension is selected right?

Actually no. See that the `cc:managedDimension` is property of the mapping resource and not the metadata resource. But you should have the option to unselect it to clear the dictionary and "disable" the mapping

Reply:
👍